toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/AuthenticatorState.kt

```diff
@@ -125,6 +135,16 @@ public sealed interface AuthenticatorState : NetworkAuthenticationChallengeHandl
      * @since 200.2.0
      */
     public fun dismissAll()
+
+    /**
+     * Revokes OAuth tokens and all credentials from the `AuthenticationManager.arcGISCredentialStore`
+     * and `AuthenticationManager.networkCredentialStore`. If an IAP credential exists in
+     * the `AuthenticationManager.arcGISCredentialStore`, this function will launch a custom tab for the user to invalidate
+     * the IAP session.
```

Comment:
Might be clearer to use bullet points here:
```suggestion
     * Revokes authentication state by:
     * - revoking OAuth tokens
     * - invalidating IAP credentials (which will launch a custom tab for the user to invalidate
     * the IAP session)
     * - removing all credentials from the `AuthenticationManager.arcGISCredentialStore`
     * - removing all credentials from the `AuthenticationManager.networkCredentialStore`
```